spring-cloud-gcp-pubsub/pom.xml

```diff
@@ -42,5 +42,34 @@
 			<artifactId>spring-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- SI channel adapters -->
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
```

Comment:
If we are sure that SI Channel Adapters should be as a part of the `spring-cloud-gcp-pubsub` module (not like `spring-cloud-gcp-integrtaion`), these dependencies for SI should be `optional`